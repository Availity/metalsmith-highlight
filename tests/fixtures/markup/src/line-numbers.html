<code class="language-javascript">
var name = "Rob";
</code>
