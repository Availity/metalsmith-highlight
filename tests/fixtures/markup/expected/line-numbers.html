<code class="language-javascript line-numbers">
<span class="token keyword">var</span> name <span class="token operator">=</span> <span class="token string">&quot;Rob&quot;</span><span class="token punctuation">;</span>
</code>
